map2loop/config.py

```diff
@@ -131,7 +133,7 @@ def update_from_legacy_file(self, file_map: dict, lower: bool = False):
             "otype": (self.structure_config, "orientation_type"),
             "dd": (self.structure_config, "dipdir_column"),
             "d": (self.structure_config, "dip_column"),
-            "sf": (self.structure_config, "description_column"),
+            "sf": (self.structure_config, "desciption_column"),
```

Comment:
This should not be reverted as it was a bug that was fixed.  "description_column" is correct.

Reply:
yep, in https://github.com/Loop3D/map2loop/pull/60/commits/450b372cfc43063a2de2eebf2b6e05458114c425